unittests/test_policies.py

```diff
@@ -591,9 +597,10 @@ def _read_timestamps(tasks):
 
 def test_concurrency_unlimited(async_runner, make_cases, make_exec_ctx):
     num_checks = 3
-    make_exec_ctx(options=max_jobs_opts(num_checks))
+    # make_exec_ctx(options=max_jobs_opts(num_checks))
```

Comment:
Why did you comment these out?

Reply:
It has no effect, since the tests run as "forced" locals and this is why I had to set manually `runner.policy._rfm_max_jobs`. Should I remove or change the test so it runs in a local partition?

Reply:
Hmm, you're right. I wonder how this worked before? Also I think that you should not set it directly on policy but do it through the configuration with the `make_exec_ctx`.

Reply:
> Also I think that you should not set it directly on policy but do it through the configuration with the make_exec_ctx.

Not sure what you mean. I set the `_rfm_max_jobs` to different values (1, 3, `num_checks`) in three different tests. Should I make 3 different fixtures or is there a way to pass an argument?